### ISSUE TITLE
Loss function issue

### ISSUE THREAD

**Issue by wengyouli** [ProRLearn/ProRLearn001#1]
Hello, I would like to ask about the loss_func and loss_func_1 in reveal-VPG-classification.py. These two loss functions are not directly defined. Are they both using the binary cross-entropy loss exemplified in bin_loss.py? How are these two loss functions specifically defined?
